### DISCUSSION TITLE
Welcome to crypto-trading-bot Discussions!

### DISCUSSION THREAD

**Discussion by amazingchow** [amazingchow/crypto-trading-bot#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Fattyfrosh001** [comment]
I need crypto trading with full details how it works

**Comment by 3APOB** [comment]
Hi, this is my first real project in Python,
a BTC/FDUSD trading bot on Binance.
https://github.com/leodeclercq/CyptoTradingBot

**Comment by AJJA0503** [comment]
Hi guys

**Comment by AJJA0503** [comment]
Can someone help me. I am an old guy in my late 40's trying to explore crypto. I need someone's help to create trading bot for bitget.

**Comment by AJJA0503** [comment]
Really appreciate your help.
Thank you
